lang/py3/setup.cfg

```diff
@@ -42,6 +42,9 @@ package_dir =
     avro = avro
 include_package_data = true
 packages = avro
+setup_requires =
+  isort
```

Comment:
This is causing cannot install issues at runtime when behind a proxy; because of ``pip -i`` cannot handle this - see https://pip.pypa.io/en/latest/reference/pip_install/#controlling-setup-requires? Can we please add a ``pyproject.toml`` to work around this, please?